frontend/test/metabase-lib/Mode.unit.spec.js

```diff
@@ -22,7 +22,7 @@ describe("Mode", () => {
         .mode();
 
     describe("forQuestion(question)", () => {
-        it("with structured query question", () => {
+        describe("with structured query question", () => {
```

Comment:
Looks like these passed before but didn't run the tests. The newer version of Jest will fail if it finds this https://github.com/facebook/jest/pull/4039.